testing/bdd_test.ts

```diff
@@ -130,15 +130,15 @@ Deno.test("global", async (t) => {
       beforeEach(beforeEachFn);
       afterEach(afterEachFn);
 
-      assertEquals(it({ name: "example 1", fn: fns[0] }), undefined);
-      assertEquals(it({ name: "example 2", fn: fns[1] }), undefined);
+      assertEquals(it({ name: "example 1", fn: fns[0]! }), undefined);
```

Comment:
There are loads of cases like this in this file. At one point I tried to be clever about it and define `fns` as `[Spy, Spy]` but it got messy very quickly. Ultimatly I decided that "worse is better" and just threw non-null assertions everywhere.

Reply:
Check out what I did in https://github.com/denoland/deno_std/pull/4084/commits/d412a7c41aedaad0b43e45662498b7d9f7609537. It should remove the need of all these non-null assertions. `as const` is powerful.

Reply:
Done. Updated all cases in mock_test.ts and bdd_test.ts to use this instead. Worth noting that bdd_test.ts uses a few callbacks, so those function types needed to be updated from `cb: (fns: Spy[]) => void,` to `cb: (fns: readonly [Spy, Spy, Spy]) => void`.